[2024-04-28T15:36:49.656+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T15:36:50.110+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_scheduled_dag_20minutes.upload_to_db scheduled__2024-04-28T15:00:00+00:00 [queued]>
[2024-04-28T15:36:50.275+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_scheduled_dag_20minutes.upload_to_db scheduled__2024-04-28T15:00:00+00:00 [queued]>
[2024-04-28T15:36:50.285+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 4
[2024-04-28T15:36:50.708+0000] {taskinstance.py:2327} INFO - Executing <Task(_PythonDecoratedOperator): upload_to_db> on 2024-04-28 15:00:00+00:00
[2024-04-28T15:36:50.764+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=89) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T15:36:50.801+0000] {standard_task_runner.py:63} INFO - Started process 91 to run task
[2024-04-28T15:36:50.773+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_scheduled_dag_20minutes', 'upload_to_db', 'scheduled__2024-04-28T15:00:00+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp5jvmumas']
[2024-04-28T15:36:50.818+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask upload_to_db
[2024-04-28T15:36:51.337+0000] {task_command.py:426} INFO - Running <TaskInstance: test_scheduled_dag_20minutes.upload_to_db scheduled__2024-04-28T15:00:00+00:00 [running]> on host 9ff320b99ca9
[2024-04-28T15:36:53.402+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='test_scheduled_dag_20minutes' AIRFLOW_CTX_TASK_ID='upload_to_db' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-28T15:00:00+00:00'
[2024-04-28T15:36:53.408+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T15:36:53.414+0000] {logging_mixin.py:188} INFO - Uploading to postgresql in schema prun_data and table order_summary
[2024-04-28T15:37:00.713+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-28T15:37:00.729+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T15:37:00.798+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=test_scheduled_dag_20minutes, task_id=upload_to_db, execution_date=20240428T150000, start_date=20240428T153650, end_date=20240428T153700
[2024-04-28T15:37:00.994+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T15:37:01.161+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T15:37:01.179+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
